components/normalizer/src/lib.rs

```diff
@@ -2559,5 +2559,5 @@ impl<'a> core::fmt::Write for IsNormalizedSinkStr<'a> {
     }
 }
 
-#[cfg(all(test, feature = "serde"))]
+#[cfg(test)]
```

Comment:
Note: I think it makes sense to keep the `feature = "serde"` rather than implicitly enabling the serde feature all the time.

Reply:
Obsolete, but I don't like feature flags applying to whole test modules, they should be on individual tests so it's clear what features the test requires.